content/en/docs/guides/opentelemetry-metrics.md

```diff
@@ -3,6 +3,8 @@ title: OpenTelemetry Metrics
 description: OpenTelemetry Metrics available in gRPC
 ---
 
+{{< youtube L6FAtc8N8Vk youtube-video >}}
```

Comment:
Move to the end of the overview section. A plain video without surrounding paragraph for context doesn't look good.

Reply:
This can be marked as resolved.